reflex/compiler/utils.py

```diff
@@ -91,8 +91,9 @@ def compile_imports(imports: imports.ImportDict) -> List[dict]:
     import_dicts = []
     for lib, fields in imports.items():
         default, rest = compile_import_statement(fields)
-        # prevent lib from being rendered on the page if all imports are NoRenderImportVar
-        if all({isinstance(f, NoRenderImportVar) for f in fields}):  # type: ignore
+
+        # prevent lib from being rendered on the page if all imports are non rendered kind
+        if all({not f.render for f in fields}):  # type: ignore
```

Comment:
probably splitting hairs at this point, but i think `not any` would read easier

```suggestion
        if not any({f.render for f in fields}):  # type: ignore
```

Reply:
I'll make the change in a future PR since there is still some work to do around the imports :+1: